src/applications/representative-appoint/pages/veteran/veteranContactMailing.js

```diff
@@ -0,0 +1,32 @@
+import {
+  addressSchema,
+  addressUI,
+  titleUI,
+  titleSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import { preparerIsVeteran } from '../../utilities/helpers';
+
+export const uiSchema = {
+  ...titleUI(
+    ({ formData }) =>
+      `${
+        preparerIsVeteran({ formData }) ? 'Your' : 'Veteran’s'
+      }  mailing address`,
+  ),
+  veteranHomeAddress: addressUI({
+    labels: {
+      militaryCheckbox: `This address is on a United States military base outside of the U.S.`,
```

Comment:
For consistency, should the claimant mailing page use this language too?

Reply:
Good catch. I made [the change](https://github.com/department-of-veterans-affairs/vets-website/pull/31492/commits/b34330e21e9eb0798976a55d6ff5e4b91f397fb6), although it may be reverted depending on design + CAIA input.